packages/bpk-component-breadcrumb/readme.md

```diff
@@ -0,0 +1,56 @@
+# bpk-component-breadcrumb
+
+> Backpack breadcrumb component.
+
+## Installation
+
+```sh
+npm install bpk-component-breadcrumb --save-dev
+```
+
+## Usage
+```js
+import React, { Component } from 'react';
+import BpkBreadcrumb, { BpkBreadcrumbItem, ARIA_CURRENT } 'bpk-component-breadcrumb';
+
+
+export default class App extends Component {
+  render() {
+    return (
+      <BpkBreadcrumb accessibilityLabel="Default breadcrumb">
+        <BpkBreadcrumbItem
+          href="/"
+          label={ARIA_CURRENT.page}
+        >
+          Home
+        </BpkBreadcrumbItem>
+        <BpkBreadcrumbItem
+          href="/page-1"
+        >
+          Page 1
+        </BpkBreadcrumbItem>
+        <BpkBreadcrumbItem active>Page 2</BpkBreadcrumbItem>
+      </BpkBreadcrumb>
+    );
+  }
+}
+
+```
+## Props
+
+### BpkBreadcrumb
+
+| Property           | PropType | Required | Default Value |
+| ------------------ | -------- | -------- | ------------- |
+| children           | node     | true     | -             |
+| accessibilityLabel | string   | true     | -             |
+
+
+### BpkBreadcrumbItem
+
+| Property           | PropType                   | Required | Default Value |
+| ------------------ | -------------------------- | -------- | ------------- |
+| children           | node                       | true     | -             |
+| href               | string                     | true     | null          |
+| active             | bool                       | false    | false         |
+| label              | oneOf('page','step','location','date', 'time','true','false'  | true     | null  |
```

Comment:
why don't we just name it `aria-current` if that is what this is being used for? It seems like this would avoid any misunderstandings about what this props is used for. 

On a side note, I think it only makes sense for it to accept `page` (maybe `step`). I don't see the breadcrumb being used for any of the other aria-current use cases.

Reply:
The reason I said `ariaCurrent` and not `aria-current` is because the React convention is to *always* use camel case, e.g. `className`.

Reply:
I tend to agree but all aria props are kebab case, and if we think about the usage of our web components, people provide aria props using kebab case, because we just forward those using `{...rest}`. 

In this case people would have to do something like this `<BreadCrumbItem ariaCurrent="label" aria-label="Label" />` which I think is not ideal

Reply:
I agree with @tiagoengel. Btw in terms of the actual values we permit ... we could jut not care and default to `string`. Think about it ... That's what React does .. it doesn't enforce strict prop types on every single prop available. Maybe we are over thinking it.

Reply:
> In this case people would have to do something like this <BreadCrumbItem ariaCurrent="page" aria-label="Label" /> which I think is not ideal

I hadn't considered that, good point.

Reply:
So are we settled for aria-current and aria-label ?